### DISCUSSION TITLE
Can we use this to generate speech from subtitle file? srt or vtt

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [neonbjb/tortoise-tts#308] [Q&A]
Hello. I want to fully replace my older video sounding by providing a manually fixed srt / vtt file

I am generating transcription by using Whisper

I am then manually fixing the file

Example video and srt file I have. So how can I use this model?

C# Expert OOP - Lecture 1: Classes, Methods, Access Modifiers, Static, IDE, DLL, Fields & Properties
https://youtu.be/s-FZ2pW6UF4

[captions (3).txt](https://github.com/snakers4/silero-models/files/10712166/captions.3.txt)

send an email from my school email furkan . gozukara at toros edu tr

also if i add my voice, do they need to be captioned? or i just provide my voice with 10 seconds split? does it have auto split feature? i have excellent sound quality recordings already

**Comment by FurkanGozukara** [comment]
if you know such paid service good quality with reasonable price for over 100 hours i can be also ok with that

like i pay 50$ and can generate 100 hours of speech from pre trained model

**Reply by satpalsr** [reply]
Hey,

@FurkanGozukara did you found a solution for this?

**Reply by FurkanGozukara** [reply]
nope all too expensive

i made a tortoise tutorial though

https://youtu.be/OiMRlqcgDL0?si=LP62LmPWcpiOnXBC

**Comment by fakerybakery** [comment]
Hi, the main issue with this is the timing. It will be hard to make Tortoise speak at the same rate as your old video, and of course if your video has faces in it the lips won't be synced. If you're trying to restore an old video, have you considered the open-source [VoiceFixer](https://github.com/voicefixer/voicefixer)? It enhances poor-quality speech and will require much less compute than Tortoise, but won't completely revoice your video.
